doc/run/repl.md

```diff
@@ -109,11 +109,12 @@ The following shortcuts ease the editing of SQL statements.
 | Keyboard Shortcut   |                        Action                          |
 |---------------------|--------------------------------------------------------|
 |<kbd>Ctrl + l</kbd>  | Clear the screen                                       |
-|<kbd>Meta + Del</kbd>| Delete the **word before** the cursor                  |
-|<kbd>Meta + d</kbd>  | Delete the **word after** the cursor                   |
-|<kbd>Ctrl + w</kbd>  | Delete the **word before** the cursor                  |
+|<kbd>Meta + Del</kbd>| Cut the **word before** the cursor to the clipboard    |
+|<kbd>Meta + d</kbd>  | Cut the **word after** the cursor to the clipboard     |
+|<kbd>Ctrl + w</kbd>  | Cut the **word before** the cursor to the clipboard    |
```

Comment:
The same with `Meta +Del` ?

Reply:
Yes, `Meta + Del` and `Ctrl + w` has the same behavior.